apps/condo/domains/banking/access/BankAccount.js

```diff
@@ -30,11 +54,17 @@ async function canReadBankAccounts ({ authentication: { item: user } }) {
  * BankAccount can be managed only by:
  * 1. Admin or support
  */
-async function canManageBankAccounts ({ authentication: { item: user } }) {
+async function canManageBankAccounts ({ authentication: { item: user }, context }) {
     if (!user) return throwAuthenticationError()
     if (user.deletedAt) return false
     if (user.isAdmin || user.isSupport) return true
 
+    if (user.type === SERVICE) {
+        if (await checkBankIntegrationsAccessRights(context, user.id, [BANK_INTEGRATION_IDS.SBBOL])) return true
```

Comment:
It seems, like after `checkBankIntegrationsAccessRights` we need perform the same checks as in read access to expect:
- either same integration context with SBBOL
- or missing integrationContext